doc/index.rst

```diff
@@ -400,3 +400,78 @@ This uses the effort command interface for the cart's degree of freedom on the r
 
   ros2 launch gz_ros2_control_demos pendulum_example_position.launch.py
   ros2 run gz_ros2_control_demos example_position
+
+
+Creating a Custom Launch File
+=============================
+
+To get started with your own launch file using `gz_ros2_control`, refer to the example launch files in the repository:
+
+`Example launch files <https://github.com/ros-controls/gz_ros2_control/tree/rolling/gz_ros2_control_demos/launch>`_
+
+Basic Elements
+--------------
+
+1. **Launching the Gazebo simulator**
+
+You can use the ``gz_sim.launch.py`` file provided by ``gz_ros2_control`` to launch Gazebo. Here's an example:
+
+.. code-block:: python
+
+    gz_sim = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(gz_sim_launch),
+        launch_arguments={
+            'gz_args': ['-r -v4 empty.sdf'],
+            'on_exit_shutdown': 'true'
+        }.items()
+    )
+
+Explanation of flags:
+
+- ``-r``: Starts the simulator immediately.
+- ``-v4``: Sets the logging verbosity to level 4.
```

Comment:
I would not advertise -v4, because the console log is spammed from gazebo and no one is looking at important warnings/errors of this plugin.